hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/StreamerUtil.java

```diff
@@ -257,6 +260,18 @@ public static HoodiePayloadConfig getPayloadConfig(Configuration conf) {
         .build();
   }
 
+  /**
+   * Returns write commit callback config with given configuration.
+   * @param conf the configuration
+   * @return a HoodieWriteCommitCallbackConfig instance
+   */
+  private static HoodieWriteCommitCallbackConfig getWriteCommitCallbackConfig(Configuration conf) {
+    Properties properties = new Properties();
+    properties.putAll(FlinkOptions.getPropertiesWithPrefix(conf.toMap(), CALLBACK_PREFIX));
+    return HoodieWriteCommitCallbackConfig.newBuilder()
+      .fromProperties(properties).build();
```

Comment:
Shouldn't `HoodieWriteConfig#setDefaults` already handle the `HoodieWriteCommitCallbackConfig` logic ?